diagnostics/WebView2.wprp

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<WindowsPerformanceRecorder Version="1.0" Comments="" Company="Microsoft Corporation" Copyright="Microsoft Corporation">
+    <Profiles>
+        <SystemProvider Id="SystemProvider_Light" Base="SystemProvider_Base">
+            <Keywords>
+                <!-- CPU -->
+                <Keyword Value="ProcessThread"/>
+                <Keyword Value="Loader"/>
+                <Keyword Value="Power"/>
+                <Keyword Value="CSwitch"/>
+                <Keyword Value="ReadyThread"/>
+                <Keyword Value="SampledProfile"/>
+                <Keyword Value="DPC"/>
+                <Keyword Value="Interrupt"/>
+                <Keyword Value="IdleStates"/>
+
+                <!-- Disk -->
+                <Keyword Value="DiskIO"/>
+                <Keyword Value="FileIO"/>
+                <Keyword Value="HardFaults"/>
+
+                <!-- Memory -->
+                <Keyword Value="MemoryInfo"/>
+                <Keyword Value="MemoryInfoWS"/>
+            </Keywords>
+            <Stacks>
+                <Stack Value="CSwitch"/>
+                <Stack Value="ReadyThread"/>
+                <Stack Value="SampledProfile"/>
```

Comment:
We might want to remove SampledProfile so that the size of the trace is small. We generally don't need CPU profiling unless it is a performance investigation.

Reply:
Done, I also removed `<Keyword Value="SampledProfile"/>` since it seems that's related too, let me know if that was wrong.

I also added `WebView2_CPU.wprp` as a second option for us to point people too in perf cases.